examples/bpi-monolith-test-suite/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "bpi-monolith-test-suite",
+  "version": "1.0.0",
```

Comment:
How are we defining versioning? Looks like different packages across the PRs are using different conventions

Reply:
Core devs meeting topic